src/mme/mme-sm.c

```diff
@@ -239,6 +241,8 @@ void mme_state_operational(ogs_fsm_t *s, mme_event_t *e)
         if (rc == OGS_OK) {
             e->enb = enb;
             e->s1ap_message = &s1ap_message;
+            const char* s1apLabel[1] = {"s1ap"};
```

Comment:
i'll need to try it out, but wouldn't it make sense to not use an array here and just define it as:
`const char* s1apLabel = "s1ap"`
really not sure here, but this relates to previous comment about declaring the list of labels ahead of time in mme-metrics.c and just re-using them (or maybe we just use pre-processor defines?)

Reply:
Good idea! That solves to problems in one go, and it seems to work. Err.. build, at least. :-)